apps/deriv-hk/pages/_document.tsx

```diff
@@ -4,6 +4,11 @@ export default function Document() {
   return (
     <Html lang="en" className="bg-background-primary-container">
       <Head>
+        <meta
+          name="viewport"
+          key="viewport"
```

Comment:
we already have this in `_app.tsx`, do we need it here as well? 
it will create a warning on dev start.